signer/apk/jar.go

```diff
@@ -6,6 +6,7 @@ import (
 	"crypto/sha256"
 	"encoding/base64"
 	"fmt"
+	"log"
```

Comment:
we don't usually log from signers. maybe gate that behind a debug statement? I'm unsure if this is an issue or not.

Reply:
`log` is only used in `isJARAligned`, which is only called from tests. It's with the application code since once everything is aligned we should validate that from the go-client.

Reply:
Added a comment to that effect.